frontend/src/templates/Header/Header.tsx

```diff
@@ -1,5 +1,79 @@
-
+import { Box, AppBar, Toolbar, IconButton, Typography, Avatar, Button } from "@mui/material";
+import {ReactComponent as Logo} from "../../assets/logo.svg";
+import {SvgIcon} from "@mui/material";
+import { useTheme } from "@mui/material";
 export function Header()
 {
-    
-}
\ No newline at end of file
+  const them = useTheme();
+    return(
+    <Box
+    maxHeight={100}>
+      <AppBar
+      >
+        <Toolbar>
+          <Box
+          flexGrow={1}  
+          display="flex"     
+          >
+        <IconButton
+            size='small'
+            edge='start'
+          >
+        <Logo
+    height={100}
+    width={400}
+    fill="white"
+    />
+            </IconButton>
+            </Box>
+            <Button variant="text" color="inherit"
+          >
+            <Typography
+            variant='h6'
+            component='div'
+            fontFamily={'Roboto'}
+          >
+            CONTACT US
+          </Typography>
+          </Button>
+          <Button variant="text" color="inherit"
+          sx={{
+            marginLeft: 10
+          }}
+          >
+          <Typography
+            variant='h6'
+            component='div'
+            fontFamily={'Roboto'}
+          >
+            ENGLISH
```

Comment:
![image](https://user-images.githubusercontent.com/72614768/224573640-34bc5b79-15ef-489d-9bb7-3ca7c61edbac.png)
This does not render as Roboto.
![image](https://user-images.githubusercontent.com/72614768/224573669-0fc12e0e-3118-47f2-be05-a66fbb2bc5af.png)